### PR TITLE
Lab#3

### PR DESCRIPTION
**Pull request by thornd3**
@jc-yap sorry its late ill finish it tonight you dont need to review it i understand

### CODE REVIEW

**Comment by jc-yap** [comment]
@thornd3 I looked through it and want to help. In A3, I think it would be helpful to write a formula for what we are trying to find, and then explain it below. You can do this through the insert->equation on the upper tab of the editor. It looks like you want to discard values at both the lower and upper tails of the distribution so you might need a sum of 2 integrals in the equation. Maybe also specify that you want to remove 5 sigma from each tail, not from the entire distribution.

**Comment by oliviatessa** [comment]
Hi Dane, 

Here are my comments on your lab 3: 
 
- Suppress your code before you publish it! Ask me or your partner if you don't know how to do this. 

P.1 
- Overall, I think you understand this problem well, but your work is hard to follow. Make sure you suppress your code and try to make your writing a little more clear. 
- Is it possible to get rid of the bad data close to the mean without affecting the statistics of the distribution? 

P.2 
- Why the Rayleigh distribution? How is the Rayleigh distribution related to a 2D Gaussian? 
- Try introducing the problem in your own words to make the report more complete and comprehensible. 

P.3 
- Since we have previously treated "more signal-like" as integration on the right tail of the distribution, it would be good to discuss why we are integrating from 0 to neg infinity for this problem. 
- Remember we can use trials factors instead of convolving 7200 times. 
- I can't find where you've found the sigma for this problem. 

Overall, it's good to see more discussion in your reports. I think if you work on making the writing more clear and formatting your report so it is easier to read, you will be in really good shape. 

Best, 
Olivia
